### DISCUSSION TITLE
switch to recent used tab

### DISCUSSION THREAD

**Discussion by bodnarlajos** [eclipse-theia/theia#13543] [Q&A]
Hi, 
I don't find how I can switch between the tabs by recently used history.

thanks in advance!

**Comment by MalcolmMielle** [comment]
I have the same question :/

**Comment by ndoschek** [comment]
Hi @bodnarlajos and @MalcolmMielle,

Thanks for bringing this up!

You're right - unfortunately, Theia currently does not support switching between editors based on recently used history, similar to the "Ctrl+Tab" behavior in VS Code. I agree that this would be a very useful enhancement. I've opened an issue to track this feature request and will keep you posted on any updates: https://github.com/eclipse-theia/theia/issues/15568

In the meantime, there is a workaround: you can customize the keybinding for the `workbench.action.quickOpen` command, which shows a list of recently opened editors. You can navigate the list using arrow keys and confirm your selection with Enter.
While it's not a true replacement for MRU-style tab switching, it provides a keyboard-driven alternative that may help in the meantime.

Thanks again for your feedback!
Nina
